README.md

```diff
@@ -16,29 +16,57 @@ Generally speaking it is very easy to run the entire application locally
 
 **Switch to using dev/staging data (optional)**
 
-To use a different Google Cloud Project, the `app/bin/configuration.dart` needs
-to be modified:
+To use a different Google Cloud Project, the `app/lib/shared/configuration.dart` needs
+to be modified. First, create a new `Configuration`, copying `Configuration._dev()`,
+but replacing `dartlang-pub-dev` with the name of a new Google Cloud project
+you will need to create:
+
+```dart
+  Configuration._localdev()
+      : projectId = '<gcloud-project>',
+        packageBucketName = '<gcloud-project>--pub-packages',
```

Comment:
While this is correct, I think the better way would be to have a static initializer method that puts the <gcloud-project> into the Strings. Would you be interested to change that part in `configuration.dart` too?

Reply:
Great idea, I will try to make those changes right away. Thanks for the feedback :)